book/src/configuration.md

```diff
@@ -23,6 +23,7 @@ To override global configuration parameters, create a `config.toml` file located
 | `idle-timeout` | Time in milliseconds since last keypress before idle timers trigger. Used for autocompletion, set to 0 for instant. | `400` |
 | `completion-trigger-len` | The min-length of word under cursor to trigger autocompletion | `2` |
 | `auto-info` | Whether to display infoboxes | `true` |
+| `file-picker-width` | File picker width, values from 0.0 to 1.0 treated as a factor | `0.5` |
```

Comment:
There's going to be an `editor.file-picker` section (https://github.com/helix-editor/helix/pull/988) so this can be a key in that sub config.

Reply:
Ok, then we have to wait #988 to be merged